Lib/sysconfig.py

```diff
@@ -95,20 +95,20 @@ def joinuser(*args):
     _INSTALL_SCHEMES |= {
         # NOTE: When modifying "purelib" scheme, update site._get_path() too.
         'nt_user': {
-            'stdlib': '{userbase}/lib/python{py_version_short}',
-            'platstdlib': '{userbase}/lib/python{py_version_short}',
-            'purelib': '{userbase}/lib/python{py_version_short}/site-packages',
-            'platlib': '{userbase}/lib/python{py_version_short}/site-packages',
-            'include': '{userbase}/include/python{py_version_short}',
+            'stdlib': '{userbase}/lib/python{py_version_plat}',
+            'platstdlib': '{userbase}/lib/python{py_version_plat}',
+            'purelib': '{userbase}/lib/python{py_version_plat}/site-packages',
+            'platlib': '{userbase}/lib/python{py_version_plat}/site-packages',
+            'include': '{userbase}/include/python{py_version_plat}',
             'scripts': '{userbase}/bin',
             'data': '{userbase}',
             },
         'posix_user': {
-            'stdlib': '{userbase}/{platlibdir}/python{py_version_short}',
```

Comment:
What's this about?

Reply:
Aren't we using posix scheme by default? That's why I changed this. But if that's not the case I'll revert.

Reply:
/me is confused :D